pkg/infra/remotecache/remotecache.go

```diff
@@ -112,7 +110,7 @@ func createClient(opts *setting.RemoteCacheOptions, sqlstore *sqlstore.SqlStore)
 // sent or received as an interface variable. Only types that will be
 // transferred as implementations of interface values need to be registered.
 // Expecting to be used only during initialization, it panics if the mapping
-// between types and names is not a bijection.
+// between types and names is not an injection.
```

Comment:
Just fixing the typo here, hope that's fine for this pull. Changes of import order are done via gofmt (which are automatic action during the save in my editor)

Reply:
Agree. But this fix is incorrect. The comment is just a copy from the wrapped function. https://github.com/golang/go/blob/go1.11.5/src/encoding/gob/type.go#L835